modules/javafx.controls/src/test/java/test/javafx/scene/control/TableCellTest.java

```diff
@@ -701,6 +702,34 @@ public void testEditCommitNullTableNullColumn() {
          cell.commitEdit("edited");
      }
 
+     @Test
+     public void testStartEditOffRangeMustNotFireStartEdit() {
+         setupForEditing();
+         int editingRow = table.getItems().size();
+         cell.updateIndex(editingRow);
+         List<CellEditEvent> events = new ArrayList<>();
+         editingColumn.addEventHandler(TableColumn.editStartEvent(), events::add);
+         cell.startEdit();
+         assertFalse("sanity: off-range cell must not be editing", cell.isEditing());
+         assertEquals("must not fire editStart", 0, events.size());
+     }
+
+     /**
+      *  Note: this is a false green until JDK-8187474 (update control editing location) is fixed
+      */
+     @Ignore("JDK-8187474")
+     @Test
+     public void testStartEditOffRangeMustNotUpdateEditingLocation() {
+         setupForEditing();
+         int editingRow = table.getItems().size();
+         cell.updateIndex(editingRow);
+         List<CellEditEvent> events = new ArrayList<>();
```

Comment:
`events` is effectively unused here, maybe you forgot the assert here?

Reply:
actually, no:  forgot to removed the unused list ;) Good catch!